src/settings/FileExtensions/ViewFileExtension.js

```diff
@@ -105,6 +153,64 @@ export class ViewFileExtension extends Component {
             >
               {record.extension}
             </Headline>
+
+            {record.metadata && (
+              <Row>
+                <Col xs={12}>
+                  <this.connectedViewMetaData metadata={record.metadata} />
```

Comment:
ViewMetaData in the current state does not suit well for the case when the record should be updated by System.